### ISSUE TITLE
Trying to Overfit Custom Dataset

### ISSUE THREAD

**Issue by bradasteiner** [ialhashim/DenseDepth#18]
Hi, I was hoping you could provide some guidance on training with a custom dataset. I am attempting to train on only a small datatset (20 images) with the goal of overfitting to those 20 images to ensure that it has a high enough capacity. 

Using the NYU model as the starting checkpoint, I have been able to train with loss steadily decreasing. However, running `test.py` on the training images gives incorrect results. (even after 500 epochs of training)

Something else I was wondering about is why the `loss` and `val_loss` have such different values if the training and validation sets are set to be identical. For example I am seeing training loss of about ~.05 compared to val_loss of 24. Could you provide any insight?

**Comment by bradasteiner** [comment]
I'm only using the `test_generator` now and it seems to be resolving the issue

**Comment by ialhashim** [comment]
This seems like an issue with the data or thr range of depth values. I usually start by making the code run in some jupyter notebook and check all value ranges for inputs and expected outputs then move the code to the training scripts.

Let me know if you are still facing issues.
